Modules/Client/Services/ClientService.php

```diff
@@ -14,11 +14,14 @@ class ClientService implements ClientServiceContract
 {
     public function index()
```

Comment:
this should accept a data array from wherever it's being called
```suggestion
    public function index($data)
```

And we should pass `request()->all()` as the data from controller. We should avoid directly using `request()` inside service file.